### PR TITLE
Basic Settings UI 

### PR DESCRIPTION
**Pull request by jorgerobles**
WIP, Added more fields, handled basic upload/download logic

### CODE REVIEW

**Comment by openhardwarecoza** [comment]
(: I've gotten spoiled with @tbfleming 's animations https://plus.google.com/u/0/101442607030198502072/posts/Z3AQ4cvth59

**Comment by tbfleming** [comment]
@jorgerobles you're getting the hang of react+redux!

@openhardwarecoza here you go:
settings

**Comment by cojarbi** [comment]
I call this LW porn.  Awesome work guys. 


--

Ariel Yahni






On Fri, Oct 28, 2016 at 8:58 AM -0500, "Todd Fleming" <notifications@github.com> wrote:












@jorgerobles you're getting the hang of react+redux!



@openhardwarecoza here you go:

File: src/components/settings.js [hunk 1]
```diff
@@ -1,19 +1,92 @@
...
-)(Settings);
+
+const mapStateToProps = (state) => {
+  return { settings: state.settings }
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    handleDownload: (settings) => {
+        /* Do I need to move this to a reducer or action?*/
```
Comment (tbfleming) [inline src/components/settings.js]: Keep it here; reducers are for changing state, not reading it.

**Comment by jorgerobles** [comment]
Cool ;)

**Pull request merged**